aces/Loris
# [Core] Add (string) cast for API electrophysiology endpoints

The results from the function `getCandID()` are casted as a string. The same thing was done to fix similar bugs for other endpoints of the API ( #5910 ). Without the cast, the candidate's ID was empty  in the JSON returned when accessing any of the pages in the testing instructions bellow. Some of them were sometimes displaying the correct information on a few pages, depending on which browser I was using, but this PR should solve the problems on both Firefox and Chrome.

#### Testing instructions 

1. Go to `https://<VM-url>.loris.ca/api/v0.0.3/candidates/300166/V1/recordings` Response should be {"Meta":{"CandID":"300166","Visit":"V1"}, ...

2. Go to `https://<VM-url>.loris.ca/api/v0.0.3/candidates/300166/V1/recordings/sub-OTT166_ses-V1_task-faceO_eeg.edf/channels`. Response should be {"Meta":{"CandID": "300166","Visit":"V1 ...

3. Go to `https://<VM-url>.loris.ca/api/v0.0.3/candidates/300166/V1/recordings/sub-OTT166_ses-V1_task-faceO_eeg.edf/events`. Response should be {"Meta":{"CandID": "300166","Visit":"V1 ...

4. Go to`https://<VM-url>.loris.ca/api/v0.0.3/candidates/300166/V1/recordings/sub-OTT166_ses-V1_task-faceO_eeg.edf/electrodes`. Response should be {"Meta":{"CandID": "300166","Visit":"V1 ...

#### Link to related issue

* Resolve #6571
